### DISCUSSION TITLE
Azure não consegue achar o gitHub

### DISCUSSION THREAD

**Discussion by JotaPeCarvalho** [balta-io/forum#233] [Tira Dúvidas] [answered]
Olá, tudo bem? Já procurei em alguns sites e não achei uma resposta que me ajude a solucionar esse problema:
![image](https://user-images.githubusercontent.com/91575096/213779847-54889dcc-9ea9-4787-9485-9c04dfaf0019.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @JotaPeCarvalho, qual curso/aula este erro ocorre? Tem mais detalhes do erro e/ou arquivo YML?
